workers/unity/Assets/Fps/Resources/Prefabs/AndroidClient/Authoritative/Player.prefab

```diff
@@ -2831,9 +2831,10 @@ MonoBehaviour:
   m_Name: 
   m_EditorClassIdentifier: 
   MovementScalar: 0.01
-  LookScalar: 0.14
+  YawMultiplier: 0.15
+  PitchMultiplier: 0.12
+  FiringLookMultiplier: 0.75
   SprintMaxAngle: 30
-  SprintDistanceThreshold: 100
```

Comment:
Is this change intentional?

Reply:
I think this is a ghost serialized property. that is, one that used to exist, got serialized, then got removed in code, but the object hadn't been reserialized until now.